go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/pivotal-cf/brokerapi/v10
 
-go 1.19
+go 1.21
```

Comment:
By looking at the previous version, and being that the go.mod go directive establishes the minimum go version required (and by the looks of it, it is also enforced since 1.21), should we declare 1.20 in the go mod whilst testing against 1.20 & 1.21?

Reply:
This might be the reason.

https://go.dev/doc/toolchain
> Before Go 1.21, Go toolchains treated the go line as an advisory requirement: if builds succeeded the toolchain assumed everything worked, and if not it printed a note about the potential version mismatch. Go 1.21 changed the go line to be a mandatory requirement instead.